android/app/src/main/kotlin/com/jabook/app/jabook/compose/feature/settings/AudioSettingsScreen.kt

```
@@ -384,95 +446,283 @@ public fun AudioSettingsScreen(
+                    preampDb = selectedEqPreset.effectivePreamp() / 100f,
+                    modifier = Modifier.padding(horizontal = contentPadding).padding(bottom = 24.dp),
+                )
+            }
         }
     }
 }
 
 @Composable
-private fun EqualizerFiveBandCard(
-    preset: EqualizerPreset,
+private fun EQCurveVisualizer(
```

detekt (1.23.8)

[warning] The function EQCurveVisualizer appears to be too complex based on Cognitive Complexity (complexity: 32). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)